deployment/provider-azure-installer.yaml

```diff
@@ -6,11 +6,14 @@ metadata:
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
+  name: csi-csi-secrets-store-provider-azure
```

Comment:
Was this change intended? All yaml changes only in `manifest_staging/deployment`

Reply:
Changes are required. Let me moved it to manifest_staging. This is also the reason to install provider resources from manifest_staging rather than from raw github source.

Reply:
Reverted all changes and adding test specific changes during test resource install.